src/radical/pilot/session.py

```diff
@@ -285,11 +285,15 @@ def close(self, cleanup=True, terminate=True, delete=None):
             logger.error("Session object already closed.")
             return
 
+        # set defaults
+        if cleanup   == None: cleanup   = True
+        if terminate == None: terminate = True
+
```

Comment:
I see the argument of the difference with mutable parameters, although I'm happy to consider those exceptions.

Anyway, I don't feel strong about it, I was really wondering whether there was a subtle difference in semantics, which you have clarified to be not the case.